hack/release.sh

```diff
@@ -137,11 +140,26 @@ function push_images() {
   login
 
   # Manager image
-  echo "pushing ${MANAGER_IMAGE_NAME}:${VERSION}"
-  docker push "${MANAGER_IMAGE_NAME}":"${VERSION}"
+  ARCH=("arm64" "amd64")
+  for arch in "${ARCH[@]}"; do
+    echo "pushing ${MANAGER_IMAGE_NAME}:${VERSION}-${arch}"
+    docker push "${MANAGER_IMAGE_NAME}":"${VERSION}-${arch}"
+    if [ "${LATEST}" ]; then
+      echo "also pushing ${MANAGER_IMAGE_NAME}:${VERSION} as latest"
+      docker push "${MANAGER_IMAGE_NAME}:latest-${arch}"
+    fi
+  done
+  docker manifest create \
+    "${MANAGER_IMAGE_NAME}:${VERSION}" \
+    --amend "${MANAGER_IMAGE_NAME}:${VERSION}-arm64" \
+    --amend "${MANAGER_IMAGE_NAME}:${VERSION}-amd64"
+  docker manifest push "${MANAGER_IMAGE_NAME}:${VERSION}"
   if [ "${LATEST}" ]; then
-    echo "also pushing ${MANAGER_IMAGE_NAME}:${VERSION} as latest"
-    docker push "${MANAGER_IMAGE_NAME}":latest
+    docker manifest create \
+      "${MANAGER_IMAGE_NAME}:latest" \
+      --amend "${MANAGER_IMAGE_NAME}:latest-arm64" \
+      --amend "${MANAGER_IMAGE_NAME}:latest-amd64"
+    docker manifest push "${MANAGER_IMAGE_NAME}:latest"
```

Comment:
I am unfamiliar with this, could you elaborate on what this command is doing??

Reply:
yes absolutely, the manifest is saying that the the image at a particular tag has more than one architectures and it points to where those are.

in this case, we are saying the latest image has two architectures and each of those correspond to the existing image tags